projects/github-actions/repo-gardening/src/tasks/gather-support-references/index.js

```diff
@@ -98,38 +100,81 @@ async function createOrUpdateComment( payload, octokit, issueReferences, issueCo
 	const { name: repo, owner } = repository;
 	const ownerLogin = owner.login;
 
-	const comment = `**Support References**
-	${ issueReferences
-		.map(
-			reference => `
-- [ ] ${ reference }`
-		)
-		.join( '' ) }
-
-	`;
-
-	const commentOpts = {
-		owner: ownerLogin,
-		repo,
-		body: comment,
-	};
-
 	const existingComment = await getListComment( issueComments );
 
 	// If there is a comment already, update it.
 	if ( existingComment !== 0 ) {
 		debug(
 			`gather-support-references: update comment ID ${ existingComment } with our new list of references.`
 		);
+
+		const {
+			data: { body: existingCommentBody },
+		} = await octokit.rest.issues.getComment( {
```

Comment:
We already have the body in `issueComments`, no need to hit the API again. Make `getListComment()` return the whole comment instead of the ID instead.

Reply:
👍 Done in cfe41688f84df3123f042f05ff86f3ef01288cde